src/libs/i18n/index.ts

```diff
@@ -4,8 +4,10 @@ import { initReactI18next } from 'react-i18next';
 import resources from './resources';
 import format from './format';
 
-export type SupportedLanguage = 'en' | 'fr';
-export const fallbackLng = 'en';
+export type SupportedLanguage = 'fr' | 'en' | 'br';
+export const fallbackLng = 'en' as SupportedLanguage;
+export const defaultLng = 'fr' as SupportedLanguage;
+export const currentLng = () => i18n.language as SupportedLanguage;
```

Comment:
Why not
```
export const currentLng = i18n.language as SupportedLanguage;
```
like the other defs?

Reply:
I find it crazy but.... even in i18next the words: locale language (or lng) are used interchangeably, I find it pretty confusing since what's defined as a language in i18next seems to support locale.
Dunno about us, I'd love if we could find a definition on when to use one or the other... @JalilArfaoui

Reply:
@lutangar  i18n.language initializes too early and if I do not restart the i18n.language when I use it (with this function) the result may be wrong for the entire app

Reply:
Alright, let's suppose will fix this issue later, but then do name this as a function instead for example:
```
getCurrentLng
```

Reply:
your right